boards/catie/zest_core_stm32g474ve/zest_core_stm32g474ve.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: zest_core_stm32g474ve
+name: Zest Core STM32G474VE
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 128
+flash: 512
+supported:
+  - pwm
+  - i2c
+  - gpio
+  - counter
+  - spi
+  - dma
+  - can
+  - rtc
+vendor: catie
```

Comment:
File format is incorrect, it should be terminated by a new-line.

Reply:
Resolved in [f5f8e15](https://github.com/catie-aq/zephyr_zest-core-stm32g474ve/pull/16/commits/f5f8e156bdccabd545913a33a613e69212b42e58).